channeldb/migration/log.go

```diff
@@ -0,0 +1,12 @@
+package migration
+
+import "github.com/btcsuite/btclog"
+
+// log is a logger that is initialized as disabled.  This means the package will
+// not perform any logging by default until a logger is set.
+var log = btclog.Disabled
+
+// UseLogger uses a specified Logger to output package logging info.
+func UseLogger(logger btclog.Logger) {
```

Comment:
Call from main `log.go`?

Reply:
done!

Reply:
In other areas we initialize a sub-sub-logger from the sub-logger. See for example `htlcswitch/log.go` initializing the payload logger.